prisma/schema.prisma

```diff
@@ -8,9 +8,10 @@ datasource db {
 }
 
 generator zod {
-  provider = "node ./lib/generator.js"
-  output   = "./generated"  
-  isGenerateSelect = true
+  provider          = "node ./lib/generator.js"
+  output            = "./generated"
+  isGenerateSelect  = false
```

Comment:
I think we should keep the example with all flags set to true.

Reply:
Flags are true now